doc/processes/release_process.md

```diff
@@ -0,0 +1,125 @@
+any OpenJDK-based JDK.  The aim is to define a set of tests, both 
+functional and stress, and 3rd party applications that should pass for 
+every release.
+
+## Platform support
+Define the platforms that are supported by a release.  The set of 
+platforms will depend on the availability of hardware at OpenJ9 and 
+AdoptOpenJDK that can be used to validate the platform meets the quality 
+bar.  The code base may support more platforms than can be validated by 
+the quality.  Such platforms will not be officially included in the release.
+
+## Performance
+OpenJ9 should avoid regressing performance from release to release.
+Each release should document the set of benchmarks and performance measures
+that will be tracked for the release.  These measures should include the
+acceptable noise margins that will be used when tracking the measure.
+Regressions may be unavoidable at times due to changes required to be compliant
+with the Java specifications, to close security holes, or to address the
+need to update / modify the architecture.  Some regressions may simply be 
+found too late in a release to address them.  
+The project may decide to trade performance - whether throughput, startup 
+or fooprint - for another important measure. 
+Any regressions should be documented as part of the release plan.
+
+Note, detecting regressions will be difficult in the open project 
+given the limited hardware resources available.  Work is being done at 
+AdoptOpenJDK to enable performance testing.
+
+# Release Process
+
+1. Create an issue to track the release and record the information required
+above, including the SHAs and tags.  This information will mostly be duplicated
+with the Eclipse release plan.
+1. Pick a candidate OpenJ9 level to mark as the initial release candidate.  This
+should be a level that has successfully passed the nightly testing builds at 
+both OpenJ9 and AdoptOpenJDK. Given these builds may target a range of SHAs, a
+candidate SHA should be taken from within that range.
+1. Branch the `openj9` & `openj9-omr` repos at the specified level.  Immediately
+tag the newly created branch with a tag of the following form: 
+`openj9-#releaseNumber#RC#candidatenumber#`.  For the `0.8` release, this would 
+result in a `openj9-0.8RC1` tag.  These branches are not intended as development
+branches and no continued support will be done on them.  They are merely points
+in time.
+1. The Extensions repos should have been branched for each of the releases.
+Update the Extensions branches to pull the tagged levels from the `openj9` 
+& `openj9-omr` release branches.
+1. Rebuild the tagged levels and ensure they pass the quality bar defined by 
+AdoptOpenJDK.
+1. Provide a window of time (a week?) for any stakeholders to highlight any 
+stopship issues with the release candidate build.  If any are found, a 
+determination can be made to either:
+	* Branch from the the `RCx` tag and apply a targetted fix, or
```

Comment:
Do we have to branch from the RCx branch or can we simply apply another change to the release branch and lay tag RC(x+1)